src/otx/algo/instance_segmentation/heads/rpn_head.py

```diff
@@ -439,23 +465,16 @@ def export_by_feat(
                 msg = "cls_score and bbox_pred should have the same size"
                 raise ValueError(msg)
             cls_score = cls_score.permute(0, 2, 3, 1)
-            if self.use_sigmoid_cls:
```

Comment:
It's just a question. Don't we have to consider a case not to use sigmoid now?

Reply:
I am not sure it it is required. We don't use softmax based solution for solving classification problem in our tasks. So, that's why I removed it